packages/language-server/package.json

```diff
@@ -16,7 +16,7 @@
   "scripts": {
     "build": "tsc",
     "dev": "astro-scripts dev \"src/**/*.ts\"",
-    "test": "cross-env TS_NODE_TRANSPILE_ONLY=true mocha --require ts-node/register \"test/**/*.ts\" --exclude \"test/**/*.d.ts\""
+    "test": "cross-env TS_NODE_TRANSPILE_ONLY=true mocha --timeout 20000 --require ts-node/register \"test/**/*.ts\" --exclude \"test/**/*.d.ts\""
```

Comment:
I do not think that it's necessary our fault that we need this, I think TypeScript is just slow and will get slower as we add more files